testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/saml/BrokerTest.java

```diff
@@ -323,6 +367,8 @@ public void testAnyNamespacePreservedInContext() throws IOException {
                 .build()
               .assertResponse(org.keycloak.testsuite.util.Matchers.statusCodeIsHC(Status.OK))
               .execute();
+        } finally {
+            clearUsers(realm);
```

Comment:
Why is this change necessary? Does this test create some users that needs to be cleared?

Reply:
Thanks, good catch :). Indeed it's not, omitted from a working state.